bundles/framework/featuredata2/event/WFSSetFilter.js

```diff
@@ -11,9 +11,10 @@ Oskari.clazz.define('Oskari.mapframework.bundle.featuredata2.event.WFSSetFilter'
      *
      */
 
-    function (geojson, filters) {
+    function (geojson, filters, allLayers) {
```

Comment:
This event is used like it was a request. I noticed it some time ago but didn't do anything about it then. But maybe we should do something about it/change it to request. I'll need to take a closer look to figure out a better approach.